src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/SoftRateLimit.swift

```diff
@@ -0,0 +1,85 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import ExposureNotification
+
+
+class SoftRateLimit {
+	// MARK: - Init
+
+	init(
+		store: Store,
+		logger: Logging? = nil
+	) {
+		self.store = store
+		self.logger = logger
+	}
+
+	func setup(configuration: RiskProvidingConfiguration) -> Bool {
+		let enoughTimeHasPassed = configuration.shouldPerformExposureDetection(
+			lastExposureDetectionDate: store.exposureDetectionDate, context: " for soft rate limit"
```

Comment:
It would be cool if you could create an enum `ExposureDetectionContext` in `RiskProvidingConfiguration` with a  case "SoftRateLimit". In this enum you add a property (description). 
Then you pass here not a `String` but a `ExposureDetectionContext`.

In the `shouldPerformExposureDetection()` you access then the enum's property to get the `String` into the log.